src/Driver/RedisSetDriver.php

```diff
@@ -2,15 +2,16 @@
 
```

Comment:
Since you are in cleaning spree. `RedisSetDriver` is only one driver without `declare(strict_types=1);`.

Reply:
didn't know about this, will add it